server.js

```diff
@@ -42,19 +42,19 @@ amqp.connect(process.env.AMPQ_ADDRESS, function(err, conn) {
       })
       .catch(next);
   });
-  app.post('/users', (req, res, next) => {
-    const { user, choiceId, questionId } = req.body;
+  app.post('/api/createForumPost', (req, res, next) => {
     var rpcInput = {
-      method: 'createUser',
-      params: [{ name: 'rob' }]
+      method: 'createForumPost',
+      params: [req.body]
```

Comment:
sanitize this by only passing what you know this method will except. You dont want a user passing bad data maliciously and it being saved into user table.

Example: `POST` `{ admin: true, email: 'rob@rob.com' }`